packages/components/src/components/common/tabs/tabs.css.ts

```diff
@@ -1,11 +1,15 @@
 import { vars } from '@/styles/theme.css';
-import { style } from '@vanilla-extract/css';
+import { createContainer, style } from '@vanilla-extract/css';
 import { recipe, type RecipeVariants } from '@vanilla-extract/recipes';
 
+export const tabsListContainer = createContainer();
+export const tabsTriggerContainer = createContainer();
```

Comment:
I think you don't have to export the list container.
And I would add the inscriptiontab styles also in here and not split it to a separate file. Then you also would not have to export the other container.